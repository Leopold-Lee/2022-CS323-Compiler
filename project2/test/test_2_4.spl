int sum(int x, int y){
    return x+y;
}

struct node{
    int a;
    float b;
};

struct node{
    int value;
    struct node next;
};

int test()
{
  int x1 = 5;
  int x4 = sum(x1,x1,x1);
  x1 = 1.2;
  return 0;
}