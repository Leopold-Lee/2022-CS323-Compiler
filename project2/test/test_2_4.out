Error type 15 at Line 10: redefine struct: node
Error type 9 at Line 18: invalid argument number for sum, expect 2, got 3
Error type 5 at Line 19: unmatching type on both sides of assignment